leda/README.md

```diff
@@ -21,15 +21,11 @@ SPDX-License-Identifier: Apache-2.0
 
 # Setup with Eclipse Leda
 
-The components of the Fleet Management blueprint fall into two categories: the *KUKSA.val Databroker*,
-*CSV Provider* and *FMS Forwarder* components are supposed to run in the vehicle, whereas the remaining
-components are supposed to run in a (cloud) back end to which the vehicle is connected via (public internet)
-networking infrastructure.
+The components of the Fleet Management blueprint fall into two categories: the *KUKSA.val Databroker*, *CSV Provider* and *FMS Forwarder* components are supposed to run in the vehicle, whereas the remaining components are supposed to run in a (cloud) back end to which the vehicle is connected via (public internet) networking infrastructure.
```

Comment:
What is the reason to make these four lines into one? (I put them into multiple lines while using a Markdown linter).

Reply:
Having the text on four lines results in the markdown being rendered into four separate (short) lines which looks a little strange e.g. in a browser window where the display is much wider than the rendered text.
So, apart from aesthetics, there is no particular reason ...

Reply:
I see that you changed this back to multiple lines, which at least in the GitHub visualization, seems not to be broken.